### ISSUE TITLE
adjust image link style in rich text (all content items): 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#354]
Users with motion, color vision, and perceptual impairments need clearer guidance when we have a linked image. Therefore, when images appear inside links anywhere in main content, they should be styled as follows. Note that the screen shots show heading text inside the same ```<a>``` element styled in accordance with #353

## initial/plain/visited:

```css
border: 3px solid #66262d;
```

![screen shot 2018-12-03 at 9 22 17 am](https://user-images.githubusercontent.com/263285/49382919-360fa380-f6dd-11e8-9189-cc3a47037120.png)

## hover/focus:

```css
border: 3px dashed blue;
```

![screen shot 2018-12-03 at 9 25 49 am](https://user-images.githubusercontent.com/263285/49383043-753df480-f6dd-11e8-815e-44c406f81da9.png)

## active

```css
border: 3px solid black
filter: sepia(1)
```

![screen shot 2018-12-03 at 9 31 51 am](https://user-images.githubusercontent.com/263285/49383418-4bd19880-f6de-11e8-8a7f-0fb2757201d3.png)

**Comment by alecpm** [comment]
This is up on staging for review.

**Comment by paregorios** [comment]
This looks good to go. Please merge to master and deploy to production.
